Sources/CoreAardvarkTests/ARKDataArchiveTests.m

```diff
@@ -264,7 +264,7 @@ - (void)test_appendArchiveOfObject_trimsCorruptedArchive;
     }];
     
     
-    [self waitForExpectationsWithTimeout:5.0 handler:nil];
+    [self waitForExpectationsWithTimeout:10.0 handler:nil];
```

Comment:
this is absurdly long 😅

Reply:
Could probably go a bit lower, but this should give us some headroom since the CI machines don't seem to be super consistent. 😅